pages/index.vue

```diff
@@ -147,15 +155,38 @@ export default defineComponent({
      * @todo 1. Send the request to update the task to the backend server.
      * @todo 2. Update the task in the dom.
      */
-    updateTask(_index, _id) {},
+    updateTask(_index, _id) {
+      let upTodo = this.msg[_index];
+      if(upTodo ==='' || upTodo === this.todos[_index].title){
```

Comment:
good check but it would not mean much if you have spaces which will exist without trimming the `upTodo`